### ISSUE TITLE
Website doesn't seem to be working

### ISSUE THREAD

**Issue by neelsomani** [Lambda-School-Labs/startup-grant-database-fe#246]
Hi there, I tried creating an account and filling out my profile, but the View Grants page isn't loading for me. It just seems to take me back to the homepage. Would it be possible for you to share a spreadsheet of the grants?

Appreciate your help, thanks!

**Comment by paulocoghi** [comment]
Exactly. The CORS is wrongly configured on the backend API, hosted on Heroku.
